### PR TITLE
Fix fallback language issue

### PR DESCRIPTION
**Pull request by hishamco**

### CODE REVIEW

File: src/Microsoft.AspNet.Localization/RequestCultureProvider.cs [hunk 1]
```diff
@@ -41,12 +42,26 @@ protected RequestCulture ValidateRequestCulture(RequestCulture requestCulture)
 
             if (Options.SupportedCultures != null && !Options.SupportedCultures.Contains(result.Culture))
             {
-                result = new RequestCulture(Options.DefaultRequestCulture.Culture, result.UICulture);
+                if(Options.SupportedCultures.Count>0)
```
Comment (Eilon) [inline src/Microsoft.AspNet.Localization/RequestCultureProvider.cs]: Please fix the formatting in this file.

File: src/Microsoft.AspNet.Localization/RequestCultureProvider.cs [hunk 2]
```diff
@@ -57,5 +72,17 @@ protected RequestCulture ValidateRequestCulture(RequestCulture requestCulture)
 
             return result;
         }
+        
+        private CultureInfo GetFallbackCulture()
```
Comment (Eilon) [inline src/Microsoft.AspNet.Localization/RequestCultureProvider.cs]: This is never called.

Comment (hishamco) [inline src/Microsoft.AspNet.Localization/RequestCultureProvider.cs]: You are right, I forget to remove it

File: src/Microsoft.AspNet.Localization/RequestCultureProvider.cs (hunk 1 above)
Comment (hishamco) [inline src/Microsoft.AspNet.Localization/RequestCultureProvider.cs]: I didn't get you!! which formatting do you mean @Eilon

**Comment by Eilon** [comment]
@DamianEdwards does this look good to you?

**Comment by hishamco** [comment]
@DamianEdwards can you please verify if this PR fix the bug?

**Comment by kirthik** [comment]
code change looks good to me but we should have tests to validate the scenario for `RequestCultureProvider` implementations.

**Comment by hishamco** [comment]
That's what I wanna post ASAP, I prepare tests for ```RequestCultureProvider``` implementations

**Comment by hishamco** [comment]
@kirthik can you validate from the scenario in issue #38 without applying my fix, seems it works fine now, i'm not sure if there's an issue in underlying Http APIs.
I wrote a unit test to validate that, I will try to push it today

**Comment by DamianEdwards** [comment]
I'm not sure this is the right fix here. This change effectively means we now fallback to whatever culture is first in the supported list, rather than fallback to the configured default culture. If we allow configuration of a default I think we should fallback to that.

**Comment by kirthik** [comment]
@hishamco closing this since @DamianEdwards said this fix is not right.

**Pull request closed without merging**